### PR TITLE
sci-electronics/fritzing: add 0.9.6

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/898070

### CODE REVIEW

File: sci-electronics/fritzing/fritzing-0.9.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	"${FILESDIR}/${PN}-0.9.4-quazip1.patch"
+	"${FILESDIR}/${PN}-0.9.6-dropping-register-keyword.patch"
+)
+
+src_prepare() {
+	xdg_src_prepare
+
+	if has_version "<dev-libs/quazip-1.0"; then
+		sed -e "/PKGCONFIG/s/quazip1-qt5/quazip/" -i phoenix.pro || die
+	fi
```
Comment (juippis) [inline sci-electronics/fritzing/fritzing-0.9.6.ebuild:56]: This is probably outdated and can be removed.

File: sci-electronics/fritzing/fritzing-0.9.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+	# Get a rid of the bundled libs
+	# Bug 412555 and
+	# https://code.google.com/p/fritzing/issues/detail?id=1898
+	rm -r src/lib/quazip/ pri/quazip.pri || die
+
+	# Fritzing doesn't need zlib
+	sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die
```
Comment (juippis) [inline sci-electronics/fritzing/fritzing-0.9.6.ebuild:64]:
```
 * SED: some expressions in the following did not cause any changes
 *     sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die;
 * no-op: -e s:-lminizip::
```

**Review by juippis — COMMENTED** [review]
Since this is a version bump and not a bug fix, a bit more is expected to land it ;) but not much. Overall seems to work on a regular system, didn't try on musl yet.

**Comment by AndrewAmmerlaan** [comment]
> The ebuild is wrong, it doesn't download the 0.9.6 of fritzing

The version is hardcoded in the ebuild. Looks like upstream has fixed their tarball naming scheme in the latest release though, so we should now be fine downloading the `${PV}.tar.gz` from their GitHub instead.

**Pull request closed without merging**